src/services/services.ts

```diff
@@ -1673,6 +1673,7 @@ namespace ts {
         }
 
         function getDocumentHighlights(fileName: string, position: number, filesToSearch: ReadonlyArray<string>): DocumentHighlights[] {
+            filesToSearch = map(filesToSearch, normalizePath);

```

Comment:
i am not sure i understand why we are doing this here.. i will come ask you in person tomorrow.